Разметка главной страницы

Nior21 opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1893379/tasks/3)

Nior21 committed:
Добавляет разметку header'а

Nior21 committed:
Добавляет разметку рекомендуемого товара в header

Nior21 committed:
Добавляет верстку main и footer разделов сайта

Nior21 committed:
Вносит небольшие исправления

Nior21 committed:
Вносит небольшие правки согласно валидатору - https://validator.w3.org/

Nior21 committed:
Вносит исправления согласно validator.w3.org

Nior21 committed:
Добавляет верные ссылки на социальные сети и главную страницу HTML Academy

Manyaka commented on img/cart_count.svg:
```diff
@@ -0,0 +1,3 @@
+<svg width="21" height="22" viewBox="0 0 21 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```
картинки должны быть названы единообразно. Оптимальнее через дефисы, как ниже
https://up.htmlacademy.ru/htmlcss/33/criteries#PROJ-03

Manyaka replied:
соотв. надо будет везде поменять, и в названиях картинок и в коде

Manyaka commented on img/quadrocopter.svg:
```diff
@@ -0,0 +1,8 @@
+<svg width="160" height="160" viewBox="0 0 160 160" fill="none" xmlns="http://www.w3.org/2000/svg">
```
наведи порядок в папке img. Свегешки отдельно, растр отдельно, хотя бы для начала

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
это не используется на этом уровне, так что убери, чтобы страница не скейлилась

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+<header class="site-header">
+
+  <nav class="user-navigation">
+    <img class="logo" src="img\logo.svg" alt="Логотип Device">
```
```suggestion
    <img class="logo" src="img/logo.svg" alt="Логотип Device.">
```

пути неправильные у картинок - надо везде поправить, и в альтах надо ставить точки на конце для интонационной паузы для скрин-ридеров - тоже поправь везде

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+
+  <nav class="user-navigation">
+    <img class="logo" src="img\logo.svg" alt="Логотип Device">
+    <h1 class="visually-hidden">Device</h1>
```
Как ты думаешь, зачем этот сайт?

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <h1 class="visually-hidden">Device</h1>
+    <form class="user-navigation search-form" action="https://echo.htmlacademy.ru/" method="POST">
+      <div class="user-navigation search-item">
+        <img class="user-navigation search-item seacrh-ico" alt="Поиск">
```
```suggestion
        <img class="user-navigation search-item search-ico" alt="" src="">
```

src - обязательный атрибут элемента img, так что надо его писать, даже если пустой,
и орфографическая ошибка в слове search - тебе надо поставить проверку орфографии (у тебя ВС кода? там есть такое, русскую и английскую проверку орфографии)

Manyaka replied:
не сделано

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <form class="user-navigation search-form" action="https://echo.htmlacademy.ru/" method="POST">
+      <div class="user-navigation search-item">
+        <img class="user-navigation search-item seacrh-ico" alt="Поиск">
+        <input class="user-navigation search-item seacrh-input" type="search" placeholder="Текст поиска">
```
у инпута должен быть обязательный label: https://up.htmlacademy.ru/htmlcss/33/criteries#HTML-05

Manyaka replied:
пока не сделано

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+        <img class="user-navigation search-item seacrh-ico" alt="Поиск">
+        <input class="user-navigation search-item seacrh-input" type="search" placeholder="Текст поиска">
+      </div>
+      <input class="user-navigation search-item submit-item" type="submit" value="Найти">
```
это переделай на кнопку, инпутами давно уже никто не пользуется там, где можно использовать кнопки

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    </form>
+
+    <img class="user-navigation sign-in-ico" src="img\account.svg" alt="Войти">
+    <a href="#" class="user-navigation sign-in">Войти</a>
```
подумай, картинка перед ссылкой на "Войти" - это контентная картинка - она должна быть в виде html тегов, или это декоративная картинка, которую ты потом добавить с помощью псевдоэлементов?

Как думаешь, чем это отличается от поиска? Можно ли на инпут повесить псевдоэлементы?

Manyaka replied:
пока не сделано

Nior21 replied:
Для таких целей используют псевдоэлементы `:before` и `:after` вместе со свойствами
`content: url('path')` или `background-image: url('path')`
Ограничение касается `<input>` и `<img>`, в них эти методы не работают.
Т.к. тут просто ссылка (тег `<a>`), то мы можем не вставлять отдельный тег img

Nior21 replied:
Пример [тут](https://codepen.io/Nior/pen/wvPWOQm)

Manyaka replied:
ну вот, ты эти картинки будешь делать псевдо-элементами, поэтому нельзя тут как в поиске использовать `<img>`. В поиске можно было, потому что на инпут псевдики не вешаются.

Так что поправь эти блоки - удали лишний тег <img>

Nior21 replied:
Так вроде сделал уже:
https://github.com/Nior21/1893379-device-33/blob/0dee4522455f29fef181b133ddc6dc3be7522b30/index.html#L31-L48

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+        <img class="user-navigation-link user-navigation-ico cart-ico" src="img\cart.svg" alt="Корзина">
+        <a class="user-navigation-link cart" href="#">Корзина</a>
+      </li>
+    </ul>
```
аналогичный вопрос про контентные и декоративные картинки

Manyaka replied:
пока не сделано

Nior21 replied:
Аналогично
https://github.com/Nior21/1893379-device-33/blob/0dee4522455f29fef181b133ddc6dc3be7522b30/index.html#L31-L48

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <ul class="main-navigation-list">
+      <li class="main-navigation-item">
+        <a class="main-navigation-link" href="catalog.html">Каталог товаров</a>
+        <img class="main-navigation-link main-navigation-ico" src="img\catalog-toggler.svg" alt="Каталог">
```
Почитай требования: https://www.figma.com/file/f8ZtnZcdWC2pSklG1ssm9K/HTML-1-%2F-%D0%94%D0%B5%D0%B2%D0%B0%D0%B9%D1%81?node-id=308%3A470 подумай, какой тег отвечает за этот блок

Nior21 replied:
Сначала логично было бы подумать что <select>, но не видел селектов которые отображаются в два столбца и более.
Возможно просто баттон и он уже попап (или как он там называется) будет открывать
Сейчас поправлю...

Manyaka replied:
да, это кнопка

Nior21 replied:
Готово. Буду ракету ставить где считаю что выполнил рекомендацию... ну так, для уверенности

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+        <a class="main-navigation-link" href="#">Контакты</a>
+      </li>
+    </ul>
+    <div class="main-navigation-background"></div>
```
Это что тут будет?

Nior21 replied:
На фоне страницы есть желтый блок, который не понятно к какому разделу будет относиться и не понятно как стилизуется. Возможно это просто фон header'а?
Если это так, то мне надо вынести за пределы header'a промо-товары

Manyaka replied:
а ты их и так вынесешь, там есть комент, где написано, что это не хедер

Manyaka replied:
щас див удали, когда будешь делать желтый блок и тебе не будет хватать дива, тогда и добавишь

Nior21 replied:
Уже сделано. Див убран

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <div class="main-navigation-background"></div>
+  </nav>
+
+  <aside class="product-preview">
```
почему тут aside - расскажи свою логику, пож

Nior21 replied:
Сейчас не на этой строке уже aside, но я пока оставил этот тег, как подходящий в структуру header/main/footer:
1) считаю что промо-продукты не относятся к тому что прямым образом объясняет суть главной страницы, реклама не является ни навигацией, ни объяснением чем занимается компания, в тексте такой информации нет и 
2) при этом сам блок не цепляется за майн, в нем нет логической зацепки за дальнейший текст (может мне так показалось),
3) если же вынести любую карточку с промо-товаром на другую страницу или как рекламу в сайдбар (ну чисто для мысленного эксперимента), то информация ничего не потеряет. Все так же есть завлекающий заголовок, описание и ссылка на оригинальную страницу товара ("подробнее").

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+
+  <aside class="product-preview">
+
+    <article class="product-preview-constructor">
```
тут тоже, почему выбрал article?

Nior21 replied:
Article рекомендуется для деления чего-то что похоже по смысловой нагрузке, но отличается так же как отдельные статьи новостей или отдельные сообщения на форуме.
В данном случае есть три отдельных карточки товара, которых может быть и больше и меньше и суть каждой из них не зависит от другой. При этом они все показывают промо-товары.

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <article class="product-preview-constructor">
+      <img class="product-preview-img" src="img/quadrocopter_with_laser.png">
+      <p class="product-preview-index">01</p>
+      <b class="product-preview-slogan">Порхает как бабочка,<br>жалит как пчела!</b>
```
```suggestion
      <p class="product-preview-slogan">Порхает как бабочка,<br>жалит как пчела!</p>
```

не надо использовать презентационные теги в html, чтобы разделять сущности: html отвечает за разметку, а за визуал и стили отвечает css

Nior21 replied:
Уже убрал.
У нас на лекции это называли "выразительностью". Я и сам не люблю использовать разные теги, т.к. пока смыслю в них мало.

Manyaka replied:
Разные теги нужно использовать, потому что в них как раз и заложен смысл, они вообще-то не просто какие-то левые буквы: они сокращения от английских слов, которые обозначают сущность, для которой эти теги сделаны! Верстать разными тегам - это верстать семантично. Ты суп ешь ложкой, а второе - вилкой, а режешь ножом. Ты выучил как пользоваться разными приборами для разной еды, значит и теги выучишь

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      <b class="product-preview-slogan">Порхает как бабочка,<br>жалит как пчела!</b>
+      <p class="product-preview-lead">Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.</p>
+      <a class="product-preview-more" href="#">Подробнее</a>
+      <div class="product-preview-row">
+        <div class="product-preview-column">
+          <p class="product-preview-content">800 м</p>
+          <p class="product-preview-content-label">Дальность полета</p>
+        </div>
+        <div class="product-preview-column">
+          <p class="product-preview-content">50 м</p>
+          <p class="product-preview-content-label">Радиус поражения</p>
+        </div>
+      </div>
```
Посмотри в сторону `dl`

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+        </div>
+      </div>
+    </article>
+
```
надо доделать остальные слайды, их можно взять на странице стайлгайда: https://www.figma.com/file/f8ZtnZcdWC2pSklG1ssm9K/HTML-1-%2F-%D0%94%D0%B5%D0%B2%D0%B0%D0%B9%D1%81?node-id=11549%3A3957

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      </div>
+    </article>
+
+    <!-- В зависимости от будущих планов по механизму переключения "крошек" этот блок требует адаптации -->
+    <nav class="product-preview-breadcrumbs">
+      <a class="product-preview-breadcrumbs-link current" href="#">
+        <img class="product-preview-breadcrumbs-img" src="img\breadcrumbs_current.svg" alt="текущий товар">
+      </a>
+      <a class="product-preview-breadcrumbs-link" href="#">
+        <img class="product-preview-breadcrumbs-img" src="img\breadcrumbs_default.svg" alt="другой товар">
+      </a>
+      <a class="product-preview-breadcrumbs-link" href="#">
+        <img class="product-preview-breadcrumbs-img" src="img\breadcrumbs_default.svg" alt="другой товар">
+      </a>
+      <br>
+      <a class="product-preview-breadcrumbs-link" href="#">
+        <img class="product-preview-control left-arrow" src="img/left_arrow_product_preview.svg" alt="листать влево">
+      </a>
+      <a class="product-preview-breadcrumbs-link" href="#">
+        <img class="product-preview-control right-arrow" src="img/right_arrow_product_preview.svg" alt="листать вправо">
+      </a>
+    </nav>
+    <!-- Конец блока с breacrumbs -->
```
это неправильно, так что можешь удалить

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+  <aside class="product-preview">
+
+    <article class="product-preview-constructor">
+      <img class="product-preview-img" src="img/quadrocopter_with_laser.png">
```
у картинки есть стрелочки - вперед/назад - надо добавить

Nior21 replied:
Вроде добавил. Если все верно, то значит готово

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      </a>
+    </nav>
+    <!-- Конец блока с breacrumbs -->
+
```
у слайдера есть переключатель - точки. Они идут на уровне слайдера, а не одного слайда. Чтобы они заехали на слайд - придется, видимо, использовать позишн абсолют, если не будешь гридом делать, ну в общем, сейчас нужно сделать просто этот блок, который будет под всеми слайдами расположен

Nior21 replied:
Я по ошибке назвал breadcrumps, не знал что такое "хлебные крошки" от слова совсем. Пока стрелки и точки сделал по свойму используя output и input type="radio". По мере понимания как это делают другие, пойму оставлять текущую схему или нет.

Вынес стрелки до формы, пусть форма отвечает за переключение картинок по реакции на нажатие радио кнопок, которые можно стилизовать по требованиям. А картинку товара уже через абсолют положу между стрелками, где заранее сделаю нужный отступ. То же самое с цифрой 01/02/03, тоже через абсолют.

Фон желтый соответсвенно будет только у header, поэтому все эту промо отправлю в main или если надо могу отдельный раздел сделать, ведь по факту он автономен. Там реклама которая может быть и на любом другом сайте размещена, только вот разметка там с наплывом, поэтому не знаю использовать aside или нет для таких целей? Само содержимое страницы не посвещено конкретно этим товарам из промо, это главная страница, где просто рассказ про компанию и ее товары. По логике промо это нечто побочное, а не мейн. Это так?

Manyaka replied:
сдавай, посмотрю

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <!-- Конец блока с breacrumbs -->
+
+  </aside>
+</header>
```
и header заканчивается до слайдера

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+
+
+<main class="site-main">
+  <nav class="site-main-catalog">
```
слайдер входит в мейн

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <ul class="site-main-catalog-list">
+      <li class="site-main-catalog-item">
+        <a class="site-main-catalog-link" href="#">
+          <img class="site-main-catalog-item-image" src="img/vr_glasses.svg" alt="Очки виртуальной реальности">
+          <h3 class="site-main-catalog-item-title">Виртуальная реальность</h3>
```
Подумать (посмотреть лайв про картинки), что это - декоративные картинки или контентные. И также, нужны ли тут заголовки - что они озаглавливают?

Manyaka replied:
соотв. сделать это для всего этого навигационного блока

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    </ul>
+  </nav>
+
+  <section class="site-main-benefits">
+    <ul class="site-main-benefits-list">
```
как только пишешь section, то сразу нужен ему заголовок, обычно второго уровня. Если его на дизе нет, то делаешь его вижуали-хидден

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <ul class="site-main-benefits-list">
+      <li class="site-main-benefits-item">
+        <h3 class="site-main-benefits-title visually-hidden">Доставка</h3>
+        <input class="site-main-benefits-button" type=button value="Доставка" action="">
```
это для чего тут?

Nior21 replied:
Убрал тут заголовки и перенес один h2 в начало section

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      </li>
+    </ul>
+    <div class="site-main-benefits-constructor">
+      <b class="site-main-benefits-title">Доставка</b>
```
Мне представляется это заголовком, тем более ниже идет текст, который этот заголовок озаглавливает. Как тебе такая идея?

Nior21 replied:
Попробовал заменить `<b>` на `<h3>`, 
возможно это хорошая идея.

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+  <section class="site-main-benefits">
+    <ul class="site-main-benefits-list">
+      <li class="site-main-benefits-item">
+        <h3 class="site-main-benefits-title visually-hidden">Доставка</h3>
```
Это не заголовки - если нечего озаглавливать, то это что-то самостоятельное. Например, это просто текст внутри лишек

Nior21 replied:
Пока поставил заголовки 3 уровня и абзацы.

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+        к вашему подъезду совершенно бесплатно! Ведь
+        мы неплохо заработаем, поднимая его на ваш этаж./p>
+      </div>
+  </section>
```
доделать остальные блоки со страницы стайлгайда: https://www.figma.com/file/f8ZtnZcdWC2pSklG1ssm9K/HTML-1-%2F-%D0%94%D0%B5%D0%B2%D0%B0%D0%B9%D1%81?node-id=11558%3A5700

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      <b class="site-main-benefits-title">Доставка</b>
+      <p class="site-main-benefits-content">Мы с удовольствием доставим ваш товар прямо
+        к вашему подъезду совершенно бесплатно! Ведь
+        мы неплохо заработаем, поднимая его на ваш этаж./p>
```
```suggestion
        мы неплохо заработаем, поднимая его на ваш этаж.</p>
```

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      </div>
+  </section>
+
+  <section class="site-main-add-offers"></section>
```
это тут что?

Nior21 replied:
```
<section class="site-main-add-offers">
    <h2 class="site-main-add-offers-title visually-hidden">Доставим редкий товар под заказ</h2>
    <img class="site-main-add-offers-image" src="" alt="Доставка ценных грузов.">
    <p class="site-main-add-offers-content">Доставим редкий товар под заказ</p>
    <img class="site-main-add-offers-more" src="" alt="Узнать подробнее.">
  </section>
```
Вот такая штука должна быть...

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+</main>
+
+
+<footer class="site-footer">
```
рано футер начал

Nior21 replied:
Попробовал исправить. Перенес в main несколько section

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+
+  <section class="site-main-add-offers"></section>
+</main>
+
```
пропустил блок с рекламным банером - Доставим редкий товар под заказ

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+
+<footer class="site-footer">
+  <section class="site-footer-about">
+    <hr class="site-footer-about">
```
это не нужно, все презентационные теги не нужны, удобнее для стилизации использовать css

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+  </section>
+
+  <section class="site-footer-contacts">
+    <hr class="site-footer-contacts">
```
не нужно

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+  <section class="site-footer-contacts">
+    <hr class="site-footer-contacts">
+    <h2 class="site-footer-contacts-title">Контакты</h2>
+    <h3 class="site-footer-contacts-heading">Интернет-магазин «Девайс»</h3>
+    <p class="site-footer-contacts-content">Вы можете забрать товар сами, заехав в наш офис.<br>Заодно сможете проверить работоспособность покупки.</p>
```
местами перепутал

Manyaka replied:
```suggestion
    <p class="site-footer-contacts-content">Вы можете забрать товар сами, заехав в наш офис. Заодно сможете проверить работоспособность покупки.</p>
```

в предыдущих параграфах в аналогичном блоке ты не использовал бры, так что и тут не надо. Ужмешь шириной потом.

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <h2 class="site-footer-contacts-title">Контакты</h2>
+    <h3 class="site-footer-contacts-heading">Интернет-магазин «Девайс»</h3>
+    <p class="site-footer-contacts-content">Вы можете забрать товар сами, заехав в наш офис.<br>Заодно сможете проверить работоспособность покупки.</p>
+    <p class="site-footer-contacts-content">Санкт-Петербург, набережная реки Карповки, 5, литера П.<br>+7 (812) 812-12-12</p>
```
```suggestion
    <p class="site-footer-contacts-content">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
    +7 (812) 812-12-12
```

сделай телефон через ссылку с телефоном

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <p class="site-footer-contacts-content">Вы можете забрать товар сами, заехав в наш офис.<br>Заодно сможете проверить работоспособность покупки.</p>
+    <p class="site-footer-contacts-content">Санкт-Петербург, набережная реки Карповки, 5, литера П.<br>+7 (812) 812-12-12</p>
+    <h3 class="site-footer-contacts-heading">Время работы</h3>
+    <p class="site-footer-contacts-content">пн—пт: с 10:00 до 22:00<br>сб—вс: с 10:00 до 19:00</p>
```
```suggestion
    <p class="site-footer-contacts-content">пн—пт: с <time>10:00</time> до 22:00<br>сб—вс: с 10:00 до 19:00</p>
       
```

ну и дальше поправь, для времени и дат есть прекрасный тег `time`

Nior21 replied:
Не уверен по поводу тега time, но для пробы сделал.
Вообще если я верно понимаю смысл этого тега, то он не помогает в случаях когда мы говорим о каком-то времени каждый день. Он либо показывает конкретную точку во времени, либо некий продолжительный кусок времени.
Мои попытки найти пример как им оформляют время работы организации не увенчались успехом.

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    <p class="site-footer-subscription-subtitle">Еженедельный дайджест с обзорами новинок,
+      длительными тест-драйвами, распаковками, а также акции, спецпредложения и вот это вот всё.</p>
+
+    <form class="site-footer-subscription-form">
```
доделай форму, у неё есть обязательные атрибуты, такие как экшен и метод

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+
+    <form class="site-footer-subscription-form">
+      <input class="site-footer-subscription-input" type="email" placeholder="Ваш e-mail">
+      <input class="site-footer-subscription-button" type="submit" value="Подписаться">
```
баттон

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      длительными тест-драйвами, распаковками, а также акции, спецпредложения и вот это вот всё.</p>
+
+    <form class="site-footer-subscription-form">
+      <input class="site-footer-subscription-input" type="email" placeholder="Ваш e-mail">
```
лейбл

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+      <input class="site-footer-subscription-button" type="submit" value="Подписаться">
+    </form>
+
+    <div class="site-footer-subscription-advantage">
+      <img src="img/checked.svg">
+      <p>Это бесплатно</p>
+    </div>
```
```suggestion
      <p>Это бесплатно</p>
```

объясни почему я оставила только один параграф?

Nior21 replied:
Потому что декоративные изображения мы не вставляем тегами, используем CSS

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    </div>
+  </section>
+
+  <nav class="site-footer-nav">
```
вот отсюда начинается футер

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+
+  <nav class="site-footer-nav">
+
+    <section class="site-footer-nav-wrapper site-footer-nav-left-wrapper">
```
не надо в футере фигачить секции, слишком маленький футер для секционного деления.

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+  <nav class="site-footer-nav">
+
+    <section class="site-footer-nav-wrapper site-footer-nav-left-wrapper">
+      <img class="site-footer-logo" src="img\logo.svg" alt="Логотип Device">
+      <a class="site-footer-nav-link adress" href="#">Санкт-Петербург, набережная реки Карповки, 5, литера П.</a>
+      <img class="site-footer-rectangle" src="" alt="-">
```
про картинки - смотри все коменты выше. Для чего адрес - это ссылка?

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+          <a class="site-footer-nav-link" href="#">Контакты</a>
+        </li>
+      </ul>
+      <ul class="site-footer-nav-list social">
+        <li class="site-footer-nav-item">
+          <a class="site-footer-nav-link" href="#">
+            Доставка
+          <img class="site-footer-nav-img" src="img/facebook.svg" alt="Facebook">
+        </a>
+        </li>
+        <li class="site-footer-nav-item">
+          <a class="site-footer-nav-link" href="#">
+            Гарантия
+          <img class="site-footer-nav-img" src="img/instagram.svg" alt="Instagram">
+        </a>
+        </li>
+        <li class="site-footer-nav-item">
+          <a class="site-footer-nav-link" href="#">
+            Контакты
+          <img class="site-footer-nav-img" src="img/twitter.svg" alt="Twitter">
+        </a>
+        </li>
+      </ul>
```
проверь текстовку

Manyaka commented on index.html:
```diff
@@ -1,12 +1,270 @@
+    </section>
+
+    <section class="site-footer-nav-wrapper site-footer-nav-right-wrapper">
+      <a class="site-footer-nav-telephone" href="tel:78128121212">Тел.: +7 (812) 812-12-12</a>
```
```suggestion
      <a class="site-footer-nav-telephone" href="tel:+78128121212">Тел.: +7 (812) 812-12-12</a>
```

Nior21 committed:
Добавляет незначительные правки

Nior21 committed:
1) Убирает не нужный метатег для работы с дисплеем;
2) заменяет c input на button кнопка search из header;
3) устраняет блок который был заготовлен для желтого фона;
4) исправляет .product-preview-slogan на тег <p> вместо <b>;
5) заменяет список .product-preview-features-list c <ul> на <dl>;
6) добавляет слайды .product-preview-constructor 02 и 03;
7) выносит стрелки .product-preview-control до формы со слайдером;
8) перемещает слайдер из раздела header в раздел aside;

Nior21 committed:
удаляет все изображения и ссылки на них

Nior21 committed:
Удаляет незавешенный catalog.html

Nior21 committed:
Удаляет каталог style с неверными стилями

Nior21 committed:
Исправляет орфографическую ошибку в .user-navigation

Nior21 commented on index.html:
```diff
@@ -1,12 +1,326 @@
+  <section class="site-main-benefits">
+    <ul class="site-main-benefits-list">
+      <li class="site-main-benefits-item">
+        <h2 class="site-main-benefits-title visually-hidden">Доставка</h2>
```
Вот в таком месте как верно создать структуру? Неужели обе строки и заголовок и label оставляем?

Manyaka replied:
Ты лучше этот комент в код вставь, то место, где тебе не понятно

Nior21 replied:
Мы обсудили в телеграмм вопрос, это уже сделано.

Nior21 committed:
Добавляет обязательный label всем input и заменяет input на button где необходимо

Nior21 committed:
Создает правки в button и input, а так же связанных с ними заголовках

Nior21 committed:
Удаляет img из 3-х .user-navigation-item

Nior21 committed:
Заменяет input на button в разделе основного меню.

Nior21 committed:
Удаляет заголовки h2 из подразделов section;
Добавляет заголовок h2 в начало section.

Nior21 committed:
Устраняет ошибку в закрывающем теге `<p>`

Nior21 committed:
Переносит 3 section из footer в main

Nior21 committed:
Исправлен class перенесенных section

Nior21 committed:
Вносит исправления в раздел "Контакты"

Nior21 committed:
Заменяет текст с телефонным номером на ссылку

Nior21 committed:
Добавляет в раздел "Время работы" тег time

Nior21 committed:
Убирает неверные сообщения в footer в social
